elements/bulbs-pdfobject/bulbs-pdfobject.js

```diff
@@ -21,7 +21,7 @@ class BulbsPDFObject extends BulbsHTMLElement {
   embedPDF () {
     let poster = this.getAttribute('poster');
     let options = {
-      fallbackLink: `<a href='[url]'><img src="${poster}"></a>`,
+      fallbackLink: `<a href='[url]'><div class="mobile-btn">Tap to explore document</div><img src="${poster}"></a>`,
```

Comment:
@jmelvnsn is the styling for this packaged with the element or responsibility of the integrating app?